modules/claims_api/spec/requests/v1/claims_request_spec.rb

```diff
@@ -22,6 +24,36 @@
     stub_mpi
   end
 
+  context 'CCG token validation' do
+    it 'raises no errors when a client is authorized for the scope they are attempting to access' do
+      with_okta_user(scopes) do |auth_header|
+        VCR.use_cassette('evss/claims/claims') do
+          allow_any_instance_of(Token).to receive(:client_credentials_token?).and_return(true)
+          allow_any_instance_of(TokenValidation::V2::Client).to receive(:token_valid?).and_return(true)
+
+          with_settings(Settings.claims_api.token_validation, api_key: 'some_value') do
+            get '/services/claims/v1/claims', params: nil, headers: request_headers.merge(auth_header)
+            expect(response.status).to eq(200)
+          end
+        end
+      end
+    end
+
+    it 'returns a 403 when a client is not authorized to access a particular claims OAuth scope' do
+      with_okta_user(scopes) do |auth_header|
+        VCR.use_cassette('evss/claims/claims') do
+          allow_any_instance_of(Token).to receive(:client_credentials_token?).and_return(true)
+          allow_any_instance_of(TokenValidation::V2::Client).to receive(:token_valid?).and_return(false)
+
+          with_settings(Settings.claims_api.token_validation, api_key: 'some_value') do
+            get '/services/claims/v1/claims', params: nil, headers: request_headers.merge(auth_header)
+            expect(response.status).to eq(403)
+          end
+        end
+      end
+    end
+  end
```

Comment:
Any chance we could move these 2 specs over into the GET POA context?  
Just feels weird that we did this work to directly support GET POA, but the specs are in the GET Claims context.

Reply:
I think I did that at first just because the majority of the logic was at the application controller level and so those checks will happen in all of the instances, but that makes sense to me, will do 👍

Reply:
Yeah, I'm totally with ya.  I was originally thinking the exact same thing.  It just looked a little out of place once we got to the end.

Reply:
Moved [here](https://github.com/department-of-veterans-affairs/vets-api/pull/8362/commits/502f778856d422ff14ba733d272e11335721cf52).